lib/web_ui/lib/src/ui/painting.dart

```diff
@@ -961,6 +961,10 @@ class PaintData {
 class Paint {
   PaintData _paintData = PaintData();
 
+  /// Constructs an empty [Paint] object with all fields initialized to
+  /// their defaults.
+  Paint();
+
```

Comment:
this should be unnecessary, FWIW. The constructor is normally implied.

Reply:
When it was omitted one of the web UI lints complained

Reply:
Remove the one in `lib/ui/painting.dart` too and the lint should go away.

Reply:
That constructor can't be removed because its used to set `_dither` to true if `enableDithering` is true. I could use cascade syntax on `_data`'s initializer, but readability suffers a little.

Reply:
I see. Ping @jonahwilliams and @yjbanov to see if the web tools (the lint) can be relaxed in this case. If not, I'm Ok with just having this dummy constructor. My experience is that there's already a lot of dummy/duplicate code in `lib/web_ui`. The ideal case would be that we don't need any duplicate code in `lib/web_ui`, and this issue would go away naturally.

Reply:
Let's keep this one. We are splitting the implementation between HTML and CanvasKit (https://github.com/flutter/engine/pull/14320), and this constructor is becoming a `factory`, which we cannot omit.